en/Building_a_Simple_Engine/Mobile_Development/03_performance_optimizations.adoc

```diff
@@ -0,0 +1,375 @@
+:pp: {plus}{plus}
+
+= Mobile Development: Performance Optimizations
+
+== Performance Optimizations for Mobile
```

Comment:
How will this overlap with the TBR best practices in the guide? (Mobile != TBR these days, but there's still an overlap.)

For TBR, a couple of the "big things" - per https://github.com/KhronosGroup/Vulkan-Guide/pull/338#issuecomment-3154166883 - are not mentioned here.

Reply:
The guide is meant to be the "encyclopedia" to the Tutorial's "class."  Thus, the guide chapter is going to have more depth and breadth by design, but the guide needs to only to touch on it to give perspective of developing for Mobile.  I'm used to thinking Mobile is equal to TBR, so my information is going to be out of date.  I'll draft what I can, please correct me where I'm wrong.  I'll ping you when the new version is up for both places.

I'll incorporate the "big things" per the comment in the guide TBR chapter.

Reply:
I added a section in the 04_rendering_approaches.adoc to cover the best practices you're mentioning here.  I also made reference to those changes here where appropriate.  I'd love to hear what you think.